data/generate_toy_data.jl

```diff
@@ -0,0 +1,43 @@
+using JSON
```

Comment:
V budoucnu by bylo ideální aby toto mělo vlastní pull request, aby se to dalo případně mergnout nezávisle.